src/sentry/profiles/task.py

```diff
@@ -324,9 +388,13 @@ def _prepare_frames_from_profile(profile: Profile) -> Tuple[List[Any], List[Any]
 
 
 def symbolicate(
-    symbolicator: Symbolicator, profile: Profile, modules: List[Any], stacktraces: List[Any]
+    symbolicator: Symbolicator,
+    profile: Profile,
+    modules: List[Any],
+    stacktraces: List[Any],
+    platform: Optional[str] = None,
```

Comment:
Looks like `platform` should not be optional here
```suggestion
    platform: str,
```